x/issuance/client/cli/tx.go

```diff
@@ -0,0 +1,177 @@
+		),
+		Args: cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			txBldr := auth.NewTxBuilderFromCLI(inBuf).WithTxEncoder(utils.GetTxEncoder(cdc))
+			tokens, err := sdk.ParseCoin(args[0])
+			if err != nil {
+				return err
+			}
+			receiver, err := sdk.AccAddressFromBech32(args[1])
+			if err != nil {
+				return err
+			}
+
+			msg := types.NewMsgIssueTokens(cliCtx.GetFromAddress(), tokens, receiver)
+			err = msg.ValidateBasic()
+			if err != nil {
+				return err
+			}
+			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
+		},
+	}
+}
+
+func getCmdRedeemTokens(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "redeem [tokens]",
+		Short: "redeem tokens",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`The asset owner redeems (burns) tokens, removing them from the circulating supply
+
+			Example:
+			$ %s tx %s redeem 20000000usdtoken
+		`, version.ClientName, types.ModuleName),
+		),
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			txBldr := auth.NewTxBuilderFromCLI(inBuf).WithTxEncoder(utils.GetTxEncoder(cdc))
+			tokens, err := sdk.ParseCoin(args[0])
+			if err != nil {
+				return err
+			}
+
+			msg := types.NewMsgRedeemTokens(cliCtx.GetFromAddress(), tokens)
+			err = msg.ValidateBasic()
+			if err != nil {
+				return err
+			}
+			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
+		},
+	}
+}
+
+func getCmdBlockAddress(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "block [address] [denom]",
+		Short: "block an address for the input denom",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`The asset owner blocks and address from holding coins of that denomination. Any tokens of the input denomination held by the address will be sent to the owner address
+
+			Example:
+			$ %s tx %s block kava15qdefkmwswysgg4qxgqpqr35k3m49pkx2jdfnw usdtoken
+		`, version.ClientName, types.ModuleName),
+		),
+		Args: cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			txBldr := auth.NewTxBuilderFromCLI(inBuf).WithTxEncoder(utils.GetTxEncoder(cdc))
+			address, err := sdk.AccAddressFromBech32(args[0])
+			if err != nil {
+				return err
+			}
+
+			err = sdk.ValidateDenom(args[1])
+			if err != nil {
+				return err
+			}
+			msg := types.NewMsgBlockAddress(cliCtx.GetFromAddress(), args[1], address)
+			err = msg.ValidateBasic()
+			if err != nil {
+				return err
+			}
+			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
+		},
+	}
+}
+
+func getCmdPauseAsset(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "pause [denom] [status]",
```

Comment:
I don't understand the status of the denom being true or false. Maybe `toggle-status [denom]` or something like that?

Reply:
maybe `kvcli tx issuance set-pause-status usdtoken true/false`. Clearer?